src/Utility.ts

```diff
@@ -20,6 +20,7 @@ export class Utility {
    */
   public async setAzureCliDynamicInstall() {
     await this.execute(`az config set extension.use_dynamic_install=yes_without_prompt`);
+    await this.execute(`az extension add --name containerapp --version 0.3.43`);
```

Comment:
Do we need to snap to a specific version of the extension? I think it would be best if we always used the latest so we didn't need to release a new version of the Action every time there's a new extension release with a fix that we need.

Reply:
Added this as a temporary fix until `--source` GA's. But I'll change this so we always install the latest version.